### DISCUSSION TITLE
My MAUI 9.0 (Preview) app crashes when playing an mp3 file

### DISCUSSION THREAD

**Discussion by behemothe** [CommunityToolkit/Maui#2204] [Q&A]
This app works with Android, but crashes with Windows when playing an MP3 file. It worked with MAUI. NET8.0. I’m with the latest version of Community.Toolkit.MAUI.MediaElement. You can’t mess around with VS2022.
The statement that crashes is: video. Play(); in the PlayMusicInProgress routine of the Ecouter.xaml.cs file. I am on Windows 11 23H2 updated AMD 64x

**Comment by VladislavAntonyuk** [comment]
We currently don’t support .net 9
